_posts/2023-09-20-intro-to-the-various-java-based-k8s-oc-tools.adoc

```diff
@@ -0,0 +1,607 @@
+---
+layout: post
+title:  "Introduction to the various Java based Kubernetes/OpenShift tools and
+their usages in WildFly related projects"
```

Comment:
I'm not sure this can be on two lines. Either that, or the title needs to also be added below like:
```
= Introduction to the various Java based Kubernetes/OpenShift tools and their usages in WildFly related projects
```
It should also likely be more brief, otherwise it will render funny.